CHANGES.md

```diff
@@ -12,6 +12,7 @@
 
 ##### Fixes :wrench:
 
+- Fixed Cesium.Viewer instantiated inside my lit component: CreditDisplay is missing its styles [#10907](https://github.com/CesiumGS/cesium/issues/10907)
```

Comment:
@rudifa Please merge in the latest `main` branch and move this to the next upcoming release- `1.104` on `04-03-2023`. Thanks!